templates/.circleci/config.yml

```diff
@@ -111,30 +112,30 @@ jobs:
     steps: # steps that comprise the `build` job
       - attach_workspace:
           at: *workspace
-      - restore_cache: # restores saved cache if no changes are detected since last run
-          keys:
-            - v1-pkg-cache-{{ checksum "go.sum" }}
-            - v1-pkg-cache-
+      - node/install-packages:
+          cache-path: "./node_modules"
+          app-dir: .
       - run:
-          name: Run unit tests
-          command: |
-            go get -u github.com/jstemmer/go-junit-report
-            mkdir -p test-reports
-            go test -v ./... | go-junit-report > test-reports/junit.xml
-
-      - save_cache: # Store cache in the /go/pkg directory
-          key: v1-pkg-cache-{{ checksum "go.sum" }}
-          paths:
-            - "/go/pkg"
-
+          name: Generating env file
+          command: cp .env.example .env
       - store_test_results:
           path: test-reports
-
       - store_artifacts:
           path: test-reports
 
       # Requires the SLACK_WEBHOOK env var to be set
       #- slack/notify-on-failure
+  lint:
```

Comment:
This job isn't used in any workflows below, so it will never be called. I would assume you would want to run it before `unit_test` if the intent is to use it for validating syntax

Reply:
Great! I'm also inexperienced with configuring CI workflows. :-)

Reply:
Addressed!